corehq/tabs/tabclasses.py

```diff
@@ -1625,6 +1626,10 @@ def sidebar_items(self):
 
         items.append((_('Manage Enterprise'), enterprise_views))
 
+        if BillingAccount._is_sms_billable_report_visible(self.domain):
```

Comment:
I'd change `_is_sms_billable_report_visible` to `is_sms_billable_report_visible` because you are accessing it outside of `BillingAccount` so it shouldn't be implicitly protected (even though it is python and works either way).